assembl/graphql/idea.py

```diff
@@ -817,6 +838,8 @@ def mutate(root, args, context, info):
         with cls.default_db.no_autoflush:
             # introducing history at every step, including thematics + questions  # noqa: E501
             thematic.copy(tombstone=True)
+
+            db = thematic.db
```

Comment:
you could have done `with cls.default_db.no_autoflush as db` too

Reply:
yeah, I know. Here, it's just code that I moved. If you prefer I can change it